hypershift-operator/controllers/nodepool/kubevirt/imagecacher.go

```diff
@@ -0,0 +1,263 @@
+package kubevirt
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-logr/logr"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	apiresource "k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/utils/pointer"
+	"kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	hyperv1 "github.com/openshift/hypershift/api/v1beta1"
+)
+
+const (
+	bootImagePVCAnnotationHash        = "hypershift.openshift.io/kubevirt-boot-image-hash"
+	bootImagePVCLabelRoleName         = "hypershift.openshift.io/kubevirt-boot-image-role"
+	bootImagePVCLabelRoleNameForPatch = "hypershift.openshift.io~1kubevirt-boot-image-role" // Can't use slashes as keys in jsonpatch. Replace them with "~1"
```

Comment:
what does the `~1` do? I haven't seen this technique before. Does the ~1 somehow turn into a `/` later on?

Reply:
This is, apparently, part of the jdon patch standard, to escape slashes.